### DISCUSSION TITLE
[1.2.1] Only one wallet is synced?

### DISCUSSION THREAD

**Discussion by tariusagi** [Chia-Network/chia-blockchain#7511] [Support]
My full node (and farmer) run with 2 wallets, one was used to created pooling plots (with its farmer key), and one was used to create NFT and receive rewards. But only the first one is synced, the second one doesn't sync at all. Is that a feature?

Running Chia 1.2.1 CLI on Ubuntu 20.04.

**Comment by rmica** [comment]
Same for me, I've imported two wallets from two node boxes and both have ID 1. This could be a reason. Didn't find any solution yet, apart from splitting back the wallets to two 1.2.x updated boxes. I'm unable to resync wallet from zero because it fails. #7171
